src/components/header.js

```diff
@@ -7,11 +7,17 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
 
 import LanguageSelector from "./language-selector"
 
+
 const Header = ({ location }) => {
   const [expanded, setExpanded] = useState("false")
-  const imageData = useStaticQuery(graphql`
+  const headerData = useStaticQuery(graphql`
```

Comment:
open to other ways of doing this, but basically I wanted to use some SiteMetadata stuff for customizing the title